the-collab-lab/tcl-15-smart-shopping-list
# Issue #14 | Welcome screen

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: Style   |

## Acceptance Criteria

- [x]  The welcome screen (the screen a user sees before they’ve created their list) looks something like the following (or equivalent, use your creativity!)

## Description

* As a team we decided to use `Bootstrap` for our CSS framework
* We added a little icon to the header and an app name ☺️

## Related Issue

closes #14

## Updates

![Screen Shot 2020-11-21 at 10 42 01 AM](https://user-images.githubusercontent.com/36022831/99874350-98724f00-2be7-11eb-929e-1a5580c7452d.png)

## Testing Steps / QA Criteria

* Pull down this branch with git pull origin `yk-dg-issue-14` and `git checkout yk-dg-issue-14`
* Run `yarn start` to launch the app to see our new welcoming page 🌟

## TODO

- Be consistent with our CSS stylesheet and files (so that we only use one file or one for each component) 
- Decide on a nicer logo
